src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/ciagmigration/CiagInductionMigrationServiceTest.kt

```diff
@@ -0,0 +1,100 @@
+import uk.gov.justice.digital.hmpps.educationandworkplanapi.app.database.jpa.entity.timeline.aValidTimelineEntity
+import uk.gov.justice.digital.hmpps.educationandworkplanapi.app.database.jpa.entity.timeline.aValidTimelineEventEntity
+
+class CiagInductionMigrationServiceTest : IntegrationTestBase() {
+
+  @Autowired
+  private lateinit var inductionMigrationService: CiagInductionMigrationService
+
+  @Autowired
+  private lateinit var inductionMigrationRepository: InductionMigrationRepository
+
+  @Autowired
+  private lateinit var wiremockService: WiremockService
+
+  @BeforeEach
+  fun resetWiremock() {
+    wiremockService.resetAllStubsAndMappings()
+  }
+
+  @Test
+  @Transactional
+  fun `should import CIAG Inductions`() {
+    // Given
+    val prisonNumber1 = aValidPrisonNumber()
+    val inductionCreatedEvent1 = aValidInductionCreatedTimelineEntity(prisonNumber = prisonNumber1)
+    val prisonNumber2 = anotherValidPrisonNumber()
+    val inductionCreatedEvent2 = aValidInductionCreatedTimelineEntity(prisonNumber = prisonNumber2)
+    timelineRepository.saveAll(listOf(inductionCreatedEvent1, inductionCreatedEvent2))
+    TestTransaction.flagForCommit()
+    TestTransaction.end()
+    TestTransaction.start()
+
+    val inductionResponse1 = aValidCiagInductionResponse(offenderId = prisonNumber1)
+    val inductionResponse2 = aValidCiagInductionResponse(offenderId = prisonNumber2)
+    wiremockService.stubGetInductionFromCiagApi(inductionResponse1)
+    wiremockService.stubGetInductionFromCiagApi(inductionResponse2)
+
+    // When
+    inductionMigrationService.migrateCiagInductions()
+
+    // Then
+    val importedInduction1 = inductionMigrationRepository.findByPrisonNumber(prisonNumber1)
+    assertThat(importedInduction1).isNotNull
+    assertThat(importedInduction1!!.prisonNumber).isEqualTo(prisonNumber1)
+    assertThat(importedInduction1.workOnRelease!!.hopingToWork).isEqualTo(HopingToWork.NOT_SURE)
+    assertThat(importedInduction1.previousQualifications!!.educationLevel).isEqualTo(SECONDARY_SCHOOL_TOOK_EXAMS)
+    assertThat(importedInduction1.previousTraining!!.trainingTypes).containsExactly(TrainingType.OTHER)
+    assertThat(importedInduction1.previousWorkExperiences!!.experiences!![0].experienceType).isEqualTo(WorkExperienceType.OTHER)
+    assertThat(importedInduction1.inPrisonInterests!!.inPrisonWorkInterests!![0].workType).isEqualTo(InPrisonWorkType.OTHER)
+    assertThat(importedInduction1.inPrisonInterests!!.inPrisonTrainingInterests!![0].trainingType).isEqualTo(InPrisonTrainingType.OTHER)
+    assertThat(importedInduction1.personalSkillsAndInterests!!.skills!![0].skillType).isEqualTo(SkillType.OTHER)
+    assertThat(importedInduction1.personalSkillsAndInterests!!.interests!![0].interestType).isEqualTo(InterestType.OTHER)
+    assertThat(importedInduction1.futureWorkInterests!!.interests!![0].workType).isEqualTo(WorkInterestType.OTHER)
```

Comment:
Normally I'd add more in-depth custom assertions, but I've chosen not to because:

1. I'd be testing the mapping of something I've already tested in `InductionMigrationMapperTest` (with the same test data builders).
2. I've manually checked the other values in the debugger and they all appear to be populated as expected.
3. We don't need long-term regression tests for future changes (which is obviously one of the main benefits of automated tests).
4. It would take me a long time to come up with various permutations. I think a dry run of importing the existing data in dev is a much better test.
5. I'm lazy! 😂